regulations-core

```diff
@@ -0,0 +1 @@
+Subproject commit 9c8324d26ab8291f9d7ac0898ab707f076199c43
```

Comment:
@KimberlyMunoz Let's remove this as it's from the testing API. I *think* it should be in the .gitignore now, so it shouldn't be a problem moving forward.